src/hooks/useSyncBrandingSetting.ts

```diff
@@ -0,0 +1,14 @@
+import { atom } from 'jotai'
+import { useUpdateAtom } from 'jotai/utils'
+import { useEffect } from 'react'
+
+export const disableBrandingAtom = atom<boolean>(false)
```

Comment:
If you have the atom in the hook file, you should only expose it through hooks. Why not a wrapper hook?

    export function useBrandingSetting() {
      return useAtomValue(disableBrandingAtom)
    }